angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-details.html

```diff
@@ -57,24 +58,24 @@
 	</div>
 	<div class="row no-margin"  ng-if='!loading && !error'>
 		<div class="col-xs-12 col-sm-4 col-lg-3 desc-item">
-			<h3 class="center">Description</h3>
-			<p class="app-description">{{::portlet.description}}</p>
+			<h3 tabindex="0" aria-label="Description" class="center">Description</h3>
```

Comment:
This is fine if it's necessary, but it makes me worried about how poorly I understand accessibility, since i would have hoped the `Description` in the content of the `<h3` element would have been good enough to get a screen reader to understand that "Description" is the level-three heading here?